src/base/storage.rs

```diff
@@ -149,6 +149,9 @@ pub unsafe trait Storage<T, R: Dim, C: Dim = U1>: RawStorage<T, R, C> {
     fn clone_owned(&self) -> Owned<T, R, C>
     where
         DefaultAllocator: Allocator<T, R, C>;
+
+    /// Cleans up the storage without calling the destructors on the contained elements.
```

Comment:
As discussed, perhaps change the name to `forget_elements`.

I think perhaps "cleans up" is a little vague. Perhaps "Drops the storage without ..." ?